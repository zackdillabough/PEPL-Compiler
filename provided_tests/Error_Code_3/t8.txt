POLY T(X, Y) = X + Y;
POLY S(R, Y, U) = R^2 + Y^2 + U;
POLY F = x + 5555;
START
INPUT A;
INPUT B;
INPUT C;

S(A, B, C);

INPUT A;
INPUT C;

Y(Y(
Y(S(A, B, C))
));


28 199 29 10 2 8 1 82 82 30
