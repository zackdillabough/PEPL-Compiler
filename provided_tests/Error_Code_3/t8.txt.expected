Error Code 3: 14 14 15 